docs/research/2026-05-13-zod-v4-breaking-changes.md

```
@@ -0,0 +1,695 @@
+  servers/exarchos-mcp/node_modules/zod/v4/classic/schemas.d.ts \
+  servers/exarchos-mcp/node_modules/zod/v4/classic/external.d.ts
+# (no output)
+```
+
+Replacement: `z.ZodType` (now defaults to `<unknown, unknown>`).
+
+Migration is a literal find-replace. Complexity: **mechanical**.
+
+**Full call-site list (`file:line`):**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)